xrdsst/controllers/diagnostics.py

```diff
@@ -0,0 +1,207 @@
+from cement import ex
+
+from xrdsst.api import DiagnosticsApi
```

Comment:
Also, it could be helpful another command for printing all the diagnostics instead of going one by one

Reply:
Added "diagnostics all" command, which calls all the methods and prints 3 tables